### DISCUSSION TITLE
Audio file not playing on some servers but works fine on others

### DISCUSSION THREAD

**Discussion by FediVideos** [mastodon/mastodon#22974] [Q&A]
Just wondering if this is an issue with Mastodon software, or a server glitch:

The audio file I've attached at https://mstdn.social/@feditips/109644044918079289 plays fine when I view the post from other instances, but it won't play on the home instance.

The download button works fine on the home instance, but the play button just turns to a pause without actually playing the file.

Do you reckon I should report this as an issue on Mastodon?

**Comment by MikeHuntington** [comment]
I'm having this same issues using Mastodon v4.1.2 - I can access the same audio files on other instances but when I press play from my home instance, nothing plays at all.
